torch/csrc/jit/passes/prepack_folding.cpp

```diff
@@ -0,0 +1,59 @@
+#include <stack>
+
+#include <torch/csrc/jit/api/module.h>
+#include <torch/csrc/jit/passes/constant_pooling.h>
+#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/passes/prepack_folding.h>
+
+namespace torch {
+namespace jit {
+
+// Must run this pass after constant folding.
+void FoldPrePackingOps(script::Module& m,
+    const PrePackingOpsFilterFn& is_foldable_op) {
+  // Since this pass can be called by quantization or other passes as well,
+  // we need to make sure we generate a unique "packed_weight_\d+"
+  // Thus static uid.
+  static int64_t uid = 0;
+
+  auto method = m.get_method("forward");
+  auto graph = method.graph();
+  std::stack<Block*> blocks_to_visit;
+  std::unordered_set<Node*> nodes_to_delete;
+  blocks_to_visit.push(graph->block());
+  std::string attr_name_base("packed_weight_");
```

Comment:
Can we make this something like `_jit_pass_packed_weight_` to reduce the chance of accidental collisions?